client/components/Map.jsx

```diff
@@ -1,20 +1,20 @@
-import React from 'react';
-import CSSModules from 'react-css-modules';
-import styles from './css/map.css';
-import GoogleMapReact from 'google-map-react';
-import GOOGLE_MAP_KEY from '../config/GoogleMap.js';
+// import React from 'react';
+// import CSSModules from 'react-css-modules';
+// import styles from './css/map.css';
+// import GoogleMapReact from 'google-map-react';
+// import GOOGLE_MAP_KEY from '../config/GoogleMap.js';
```

Comment:
just wondering, why are these lines commented out?

Reply:
Not using google maps because its config and api key should be private and is included in gitignore.

Reply:
ok, thanks for clarifying